config/routes.rb

```diff
@@ -9,6 +9,8 @@
           get :lock, on: :member, action: :show, controller: 'dlmlocks'
           put :lock, on: :member, action: :acquire, controller: 'dlmlocks'
           delete :lock, on: :member, action: :release, controller: 'dlmlocks'
+
+          get :events, on: :member, action: :events, controller: 'dlmlocks'
```

Comment:
Please use
```
member do
  get :events
end
```
as this is cleaner. The explicit versions were only needed because we mapped different paths to the same action.